src/java.base/share/classes/java/lang/StackStreamFactory.java

```diff
@@ -88,7 +87,7 @@ private StackStreamFactory() {}
      * VM built-in backtrace filled in Throwable with the StackWalker.
```

Comment:
This leftover comment used to belong to the `useNewThrowable` field which was removed in merge commit 4db74fef1e73c008d044c681d26c7444b2245316 back in 2017.

You may consider removing it, since it now seems to belong to the `isDebug` field, which is not the case.

Reply:
No time like the present